### DISCUSSION TITLE
Contribution Avaliable

### DISCUSSION THREAD

**Discussion by humamalamin** [dzikirqu/dzikirqu-android#15] [Q&A] [answered]
Assalamualaikum, saya backend developer. apakah bisa ikut kontribusi?

**Comment by ImBIOS** [comment, marked as answer]
Wa'alaikumussalam, currently hanya baru android yang open source karena project lain sedang under maintenance. Insyaa Allah akan ada beberapa project lain yang akan go public. We just responding to muslim community request.

Harapannya semua bisa open source, karena tidak ada hal yang perlu kami sembunyikan dari user dan komunitas ini.

**Comment by ImBIOS** [comment]
Bahasa backend apa yang dipakai @humamalamin ?

**Reply by humamalamin** [reply]
saya biasa nya pake PHP atau Golang mas

**Reply by ImBIOS** [reply]
Wah, great! I can't say anything because it is not launched yet. Hope the launch day is near.
